library/validation/validation_script.js.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/*If the validation (new) that uses the validate.js library was set on "on" in globals include the following libraries*/
+if($GLOBALS['new_validate']) {
+?>
+    <script type="text/javascript" src="<?php echo $GLOBALS['rootdir'] ?>/../library/js/vendors/moment.js"></script>
+    <script type="text/javascript" src="<?php echo $GLOBALS['rootdir'] ?>/../library/js/vendors/validate/validate.js"></script>
+    <script type="text/javascript" src="<?php echo $GLOBALS['rootdir'] ?>/../library/js/vendors/validate/validate_extend.js"></script>
+<?php
+ }
+?>
+
+<script language='JavaScript'>
+    <?php /*Added 2 parameters to the already existing submitme form*/
+    /*new validate: Use the new validation library (comes from globals)*/
+    /*e: event*/
+    /*form id: used to get the validation rules*/?>
+
+    function submitme(new_validate,e,form_id) {
+
+    //Use the old validation script if no parameter sent (backward compatibility)
+    //if we want to use the "old" validate function (set in globals) the validate function that will be called is the one that
+    // was on code up to today (the validat library was not loaded ( look at the top of this file)
```

Comment:
No, because the validation library is not loaded, it loads only if the "new validation is used"